firstvoices/backend/urls.py

```diff
@@ -9,34 +9,36 @@
 from backend.views.data_views import SitesDataViewSet
 from backend.views.dictionary_views import DictionaryViewSet
 from backend.views.parts_of_speech_views import PartsOfSpeechViewSet
+from backend.views.person_views import PersonViewSet
 from backend.views.search.search_views import SearchViewSet
 from backend.views.sites_views import MySitesViewSet, SiteViewSet
 from backend.views.user import UserViewSet
 from backend.views.word_of_the_day_views import WordOfTheDayView
 
 # app-level APIs
 ROUTER = DefaultRouter(trailing_slash=True)
-ROUTER.register(r"user", UserViewSet, basename=r"user")
-ROUTER.register(r"search", SearchViewSet, basename="search")
-ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="partofspeech")
 ROUTER.register(r"my-sites", MySitesViewSet, basename="my-sites")
+ROUTER.register(r"parts-of-speech", PartsOfSpeechViewSet, basename="partofspeech")
+ROUTER.register(r"search", SearchViewSet, basename="search")
+ROUTER.register(r"user", UserViewSet, basename=r"user")
 ROUTER.register(r"sites", SiteViewSet, basename="site")
 
```

Comment:
I alphabetized these, except for leaving "sites" at the bottom. Open to other opinions; I was just finding it hard to find things in the long list. (Exciting that it is now long enough that this is a problem!)

Reply:
+1 for alphabetization